tests/test_result/test_result_map.py

```diff
@@ -4,20 +4,20 @@
 
 
 def test_map_success():
-    """Ensures that Failure identity works for Success container."""
+    """Ensures that Success is mappable."""
     assert Success(5).map(str) == Success('5')
 
 
 def test_map_failure():
-    """Ensures that Failure identity works for Success container."""
-    assert Failure(5).map(str) == Failure(5)
+    """Ensures that Failure is mappable."""
+    assert Failure(5).map_failure(str) == Failure('5')
```

Comment:
We still need to fix that `.map()` does nothing for `Failure` and `.map_failure()` does nothing for `Success`